biblib/views/base_view.py

```diff
@@ -15,14 +17,16 @@
 from sqlalchemy.orm.exc import NoResultFound
 from sqlalchemy import Boolean
 from ..biblib_exceptions import BackendIntegrityError, PermissionDeniedError
-from ..utils import uniquify
+from ..utils import uniquify, err
 from ..emails import Email
 
 class BaseView(Resource):
     """
     A base view class to keep a single version of common functions used between
     all of the views.
     """
+    #default permissions for write_access()
+    write_allowed = ['write', 'admin', 'owner']
```

Comment:
It's defined here because several functions use write access and most of them need this version of writes_allowed, but one version needs slightly different permissions. Everything else about the function is the same.

Reply:
`TransferView` needs different permissions so I defined them as a part of `BaseView` and the overwrite it in `TransferView`